kubernetes/assets/dashboards/kubernetes_pods.json

```diff
@@ -1,805 +1,897 @@
+                "type": "timeseries",
                 "requests": [
-                    {
-                        "q": "top(sum:kubernetes.memory.usage{$scope,$namespace,$deployment,!pod_name:no_pod,$cluster,$statefulset,$daemonset,$job} by {pod_name}, 10, 'mean', 'desc')"
-                    }
+                  {
+                    "formulas": [
+                      { "formula": "query1", "alias": "Pods Running" }
+                    ],
+                    "response_format": "timeseries",
+                    "on_right_yaxis": false,
+                    "queries": [
+                      {
+                        "query": "sum:kubernetes.pods.running{$scope,$cluster,$namespace,$deployment,$statefulset,$daemonset,$job}",
+                        "data_source": "metrics",
+                        "name": "query1"
+                      }
+                    ],
+                    "style": {
+                      "palette": "green",
+                      "line_type": "solid",
+                      "line_width": "normal"
+                    },
+                    "display_type": "line"
+                  }
                 ],
-                "custom_links": [],
-                "title": "Most memory-intensive pods",
+                "yaxis": {
+                  "scale": "linear",
+                  "label": "",
+                  "include_zero": true,
+                  "min": "auto",
+                  "max": "auto"
+                },
+                "markers": []
+              },
+              "layout": { "x": 0, "y": 0, "width": 4, "height": 2 }
+            },
+            {
+              "id": 2463813939398348,
+              "definition": {
+                "title": "Pods Running (changed weekly)",
                 "title_size": "16",
                 "title_align": "left",
-                "time": {
-                    "live_span": "4h"
-                }
+                "type": "change",
+                "custom_links": [],
+                "requests": [
+                  {
+                    "q": "sum:kubernetes.pods.running{$scope,$namespace,$cluster,$deployment,$statefulset,$daemonset,$job} by {kube_namespace,kube_deployment}",
+                    "order_by": "change",
+                    "order_dir": "desc",
+                    "compare_to": "week_before",
+                    "increase_good": true,
+                    "change_type": "absolute"
+                  }
+                ]
+              },
+              "layout": { "x": 4, "y": 0, "width": 2, "height": 6 }
             },
-            "layout": {
-                "x": 29,
-                "y": 52,
-                "width": 28,
-                "height": 24
-            }
-        },
-        {
-            "id": 2,
-            "definition": {
+            {
+              "id": 7135389228788322,
+              "definition": {
+                "title": "Pods in Bad Phase by Namespaces",
```

Comment:
This widget previously was in red since they denoted not successful. Was the change to green intentional?

Reply:
yes, it was changed to match the group header color